src/commands/createRepo/GitignoreListStep.ts

```diff
@@ -19,11 +19,9 @@ export class GitignoreListStep extends AzureWizardPromptStep<IStaticWebAppWizard
         const gitignorePath: string = join(fsPath, '.gitignore');
         const files: string[] = (await fse.readdir(fsPath))
             .filter(name => name !== '.git');
-        const placeHolder: string = localize('ignore', 'Select which files should be included in the repository.');
+        const placeHolder: string = localize('ignore', 'Select which files should be ignored in the repository.');
```

Comment:
So the only reason I'm hesitant on this change is because the publish experience from the SCM client in VS Code asks for which files to include rather than ignore.  So if users have used that experience before, this might be confusing.

Anthony's request was to select everything by default, but this seems like a good alternative.  I think I'd like a second opinion: @ejizba @fiveisprime

Reply:
Regardless of what's better ("ignored" or "included"), I don't think you can flip it after it's already been released. If we were convinced one was markedly better than the other (I'm not), we could maybe come up with a transition plan, but I think it would be messy.

That being said, I think something _does_ need to change. Users should be able to click "enter" for every prompt in our wizard and it should "just work".

Here's a few options I like better:
1. Get rid of the prompt and just use some sort of default `.gitignore`. Nathan modeled this experience after the SCM client, but maybe this prompt only makes sense in the context of the SCM wizard and not the SWA wizard
2. Go with Anthony's request and select stuff by default. Doesn't have to be everything - you could be smart and uncheck some common folders people would ignore. One benefit of checking things by default is I think it makes it more obvious that we _did_ have a default and they can just click "enter"

Reply:
I like option 2. It seems better to give people the option to ignore specific files if they want